plaid/accounts.go

```diff
@@ -0,0 +1,102 @@
+package plaid
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+type Account struct {
+	ID      string `json:"account_id"`
+	Balance struct {
+		Available              float64 `json:"available"`
+		Current                float64 `json:"current"`
+		Limit                  float64 `json:"limit"`
+		ISOCurrencyCode        string  `json:"iso_currency_code"`
+		UnofficialCurrencyCode string  `json:"unofficial_currency_code"`
+	} `json:"balances"`
+	Mask         string `json:"mask"`
+	Name         string `json:"name"`
+	OfficialName string `json:"official_name"`
+	Subtype      string `json:"subtype"`
+	Type         string `json:"type"`
+}
+
+type AccountNumber struct {
+	Account     string `json:"account"`
+	AccountID   string `json:"account_id"`
+	Routing     string `json:"routing"`
+	WireRouting string `json:"wire_routing"`
+}
+
+type accountRequestJsonOptions struct {
+	AccountIDs []string `json:"account_ids"`
+}
+
+type accountRequestJson struct {
+	ClientID    string                    `json:"client_id"`
+	Secret      string                    `json:"secret"`
+	AccessToken string                    `json:"access_token"`
+	Options     accountRequestJsonOptions `json:"options"`
```

Comment:
we should add `omitempty` to `json:"options,omitempty"` since `Options` can be omitted in this request

Reply:
fixed!